test/test.html

```diff
@@ -38,6 +38,7 @@
   <script src="unit/core/core.js" type="text/javascript" ></script>
   <!--<script src="unit/core/2d_primitives.js" type="text/javascript" ></script>-->
   <script src="unit/core/curves.js" type="text/javascript" ></script>
+  <script src="unit/core/error_helpers.js" type="text/javascript" ></script>
```

Comment:
Note that I'm not adding this suite to `test-minified.html` because the friendly error help isn't supposed to be part of minified (optimized) builds.